backend/core/src/application-flagged-sets/entities/application-flagged-set.entity.ts

```diff
@@ -32,11 +32,6 @@ export class ApplicationFlaggedSet extends AbstractEntity {
   @Type(() => User)
   resolvingUser: User
 
-  @Column({ enum: FlaggedSetStatus, nullable: false, default: FlaggedSetStatus.flagged })
```

Comment:
Unrelated but above do you mind typing `rule` as the enum?

Reply:
Hey @emilyjablonski I'm a lil confused on this comment, that line is removing the column from application flagged-set entity?

Reply:
It's a few lines above :) I can handle in a separate PR as well, just a typing issue I ran into